web/styles/portico/portico.css

```diff
@@ -587,7 +587,7 @@ input.text-error {
     }
 
     tbody > tr:nth-child(odd) > td {
-        background-color: hsl(0deg 0% 98%);
+        background-color: var(--color-background-modal);
```

Comment:
The portico does not import CSS variables (and also does not do dark theme). So this change should be removed.

Reply:
Okay removed this change.